SimPEG/electromagnetics/time_domain/sources.py

```diff
@@ -4,19 +4,18 @@
 from geoana.em.static import CircularLoopWholeSpace, MagneticDipoleWholeSpace
 from scipy.constants import mu_0
 
+from ...utils import Zero, sdiag
 from ...utils.code_utils import (
     deprecate_property,
-    validate_float,
-    validate_string,
-    validate_type,
-    validate_ndarray_with_shape,
-    validate_location_property,
     validate_callable,
     validate_direction,
+    validate_float,
     validate_integer,
+    validate_location_property,
+    validate_ndarray_with_shape,
+    validate_string,
+    validate_type,
 )
-
-from ...utils import sdiag, Zero
```

Comment:
Do you have something in your IDE that automatically reorders the imports alphabetically?

Reply:
I do, yes. I think it works alphabetically but also type of import (whether a package, subplackage, or local import)